website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -44,19 +44,24 @@ Please read through the entire procedure before beginning the upgrade. There are
 
 2. Review for breaking changes between v0.33 and v0.37: If you are already running Karpenter v0.37.x, you can skip this step. If you are running an earlier Karpenter version, you need to review the [Upgrade Guide]({{<ref "upgrade-guide#upgrading-to-0320" >}}) for each minor release.
 
-3. Set environment variables for your cluster:
+3. Set environment variables for your cluster to upgrade to the latest patch version of the current Karpenter version you're running on:
```

Comment:
Do we need to give more info about why we are telling users to do this? Otherwise, I could see questions about why I wouldn't just go straight from my current version to 1.0.0

Reply:
I added a note at the beginning of the upgrade guide